python/ray/includes/metric.pxi

```diff
@@ -0,0 +1,100 @@
+from ray.includes.metric cimport (
+    CCount,
+    CGauge,
+    CHistogram,
+    CTagKey,
+    CSum,
+    CMetric,
+)
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string as c_string
+from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector as c_vector
+
+cdef class TagKey:
+    cdef c_string __name
+    def __init__(self, name):
+      self.__name = name.encode("ascii")
+      CTagKey.Register(self.__name)
+    
+    def name(self):
+      return self.__name
+
+cdef class Metric:
```

Comment:
We eventually need docstring to all of them, but it is out of scope for now.

Reply:
It's only a wrapper file. So I add a brief introduction of metrics in metric.py.

Reply:
We still need a docstring for each individual class for auto-documentation (the docstring you wrote will only be useful to people reading the code).